modules/apply-performance-profile-hosted-cluster.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-tuning-operator.adoc
+// * hosted_control_planes/hcp-machine-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="apply-performance-profile-hosted-cluster_{context}"]
+= Configuring low latency tuning in a hosted cluster
+
+To set low latency with the performance profile on the nodes in your hosted cluster, you can use the Node Tuning Operator. In {hcp}, you can configure low latency tuning by creating config maps that contain `Tuned` objects and referencing those config maps in your node pools. The tuned object in this case is a `PerformanceProfile` object that defines the performance profile you want to apply to the nodes in a node pool.
```

Comment:
Does the way `{hcp}` is written is on purpose?

Reply:
that is an variable which will be populated when the docs build